doc/fuzzing.md

```diff
@@ -153,13 +153,18 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CXX_FLAG `-fuse-ld=lld`.
+
 Full configuration step that was tested on macOS with `brew` installed `llvm`:
 
 ```sh
+$ brew install llvm
+$ brew install lld
```

Comment:
we can likely do this in a single command:
```suggestion
$ brew install llvm lld
```


We also likely need to remove the command from https://github.com/bitcoin/bitcoin/blob/master/doc/fuzzing.md?plain=1#L150 now

Reply:
Taken